selfdrive/car/toyota/interface.py

```diff
@@ -120,95 +116,109 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], disabl
       ret.steerRatio = 14.8  # Found at https://pressroom.toyota.com/releases/2016+avalon+product+specs.download
       tire_stiffness_factor = 0.7983
       ret.mass = 3505. * CV.LB_TO_KG + STD_CARGO_KG  # mean between normal and hybrid
-      set_lat_tune(ret.lateralTuning, LatTunes.PID_H)
+      # TODO override until there is enough data
+      if candidate == CAR.AVALON:
+        torque_params = CarInterfaceBase.get_torque_params(CAR.AVALON_2019)
 
     elif candidate in (CAR.RAV4_TSS2, CAR.RAV4_TSS2_2022, CAR.RAV4H_TSS2, CAR.RAV4H_TSS2_2022):
       stop_and_go = True
       ret.wheelbase = 2.68986
       ret.steerRatio = 14.3
       tire_stiffness_factor = 0.7933
       ret.mass = 3585. * CV.LB_TO_KG + STD_CARGO_KG  # Average between ICE and Hybrid
-      set_lat_tune(ret.lateralTuning, LatTunes.PID_D)
```

Comment:
I think you are missing a default tune here.  For example, what tune would `CAR.RAV4H_TSS2` end up with?

Reply:
Apologies, I see that is what line 34 is doing.